client/trino-client/src/main/java/io/trino/client/StageStats.java

```diff
@@ -57,6 +59,8 @@ public StageStats(
             @JsonProperty("processedRows") long processedRows,
             @JsonProperty("processedBytes") long processedBytes,
             @JsonProperty("physicalInputBytes") long physicalInputBytes,
+            @JsonProperty("failedTasks") int failedTasks,
+            @JsonProperty("coordinatorOnly") boolean coordinatorOnly,
```

Comment:
those changes do not seem tests specific

Reply:
They are not, but I need them for tests. Not sure if it makes sense to extract them into a separate commit, as then it would be harder to understand why do we need it.